x-pack/plugins/security_solution/server/lib/detection_engine/rule_monitoring/logic/detection_engine_health/detection_engine_health_client.ts

```diff
@@ -134,7 +135,12 @@ export const createDetectionEngineHealthClient = (
         'IDetectionEngineHealthClient.installAssetsForMonitoringHealth',
         async () => {
           try {
-            await installAssetsForRuleMonitoring(savedObjectsImporter, logger, currentSpaceId);
+            await installAssetsForRuleMonitoring(
+              savedObjectsImporter,
+              logger,
+              currentSpaceId,
+              savedObjectsClient
```

Comment:
So instead of passing a `savedObjectsClient` we'd need to pass some sort of `securitySolutionTagsClient` which would have the `ensureSecuritySolutionTagIsCreated` method. Or something like that.